source/agora/consensus/Quorum.d

```diff
@@ -60,17 +60,20 @@ private enum double THRESHOLD = 0.80;
 /*******************************************************************************
 
     Build the quorum configuration for the given public key and the registered
-    enrollment keys.
+    enrollment keys. The random seed is used to introduce partial randomness
+    to the quorum assignment of each node.
 
     Params:
         key = the key of the node for which to generate the quorum
         utxo_keys = the list of UTXO keys of all the active enrollments
         finder = UTXO finder delegate
+        rand_seed = the random seed
 
 *******************************************************************************/
 
 public QuorumConfig buildQuorumConfig ( const ref PublicKey key,
-    in Hash[] utxo_keys, scope UTXOFinder finder ) @safe nothrow
+    in Hash[] utxo_keys, scope UTXOFinder finder, const ref Hash rand_seed ) 
```

Comment:
`foo (int bar)` not `foo ( int bar )` but that was already the case before this PR

Reply:
dfmt bro.